partiql-plan/src/main/resources/partiql_plan.ion

```diff
@@ -8,15 +8,22 @@ imports::{
 }
 
 parti_q_l_plan::{
-  globals:    list::[global],       // (globals ...)
+  catalogs:   list::[catalog],      // (catalogs ...)
   statement:  statement,            // (statement ...)
 }
 
-// Globals
-
-global::{
-  path: '.identifier.qualified',
-  type: static_type,
+// Represent an instance of a database.
+// - Currently, `values` represents all values from this catalog to be used in this plan.
+// - Eventually, TODO functions may be resolved to a specific namespace within a catalog.
+catalog::{
+  name: string,
+  values: list::[value],
+  _: [
+    value::{
+      path: list::[string], // This should NOT be empty.
+      type: static_type
+    }
+  ]
```

Comment:
```suggestion
catalog::{
  name: string,
  symbols: list::[symbol],
}
  
symbol::[
    value::{
        path: list::[string], // This should NOT be empty.
        type: static_type
    }
    // TODO fn
    _::[
      ref::{
         catalog: int,
         symbol: int,
      }
    ]
]
```



This structure also aligns with https://en.wikipedia.org/wiki/Symbol_table

> The minimum information contained in a symbol table used by a translator and [intermediate representation](https://en.wikipedia.org/wiki/Intermediate_representation) (IR) includes the symbol's name and its location or address.

Reply:
I like the use of `ref` a lot. Modified slightly for the nesting, but just pushed.